src/components/DayPicker.jsx

```diff
@@ -621,7 +631,7 @@ export default class DayPicker extends React.Component {
 
     return (
       <div
-        className="DayPicker__week-header"
+        className={cx("DayPicker__week-header", { "DayPicker__week-header--rtl": isRTL })} // sag1v rtl
```

Comment:
these double quotes need to be single quotes when in a JS context (previously it was in a jsx context, which requires double)